#! /bin/sh /usr/share/dpatch/dpatch-run
## 31_manpages_lintian.dpatch by Dirk Eddelbuettel <edd@debian.org>
##
## DP: Fixes lintian errors  with manpages

@DPATCH@

diff -urNad openmpi-1.2.3.orig/ompi/mpi/man/man3/MPI_Type_create_darray.3 openmpi-1.2.3/ompi/mpi/man/man3/MPI_Type_create_darray.3
--- openmpi-1.2.3.orig/ompi/mpi/man/man3/MPI_Type_create_darray.3	2007-02-16 12:20:45.000000000 -0600
+++ openmpi-1.2.3/ompi/mpi/man/man3/MPI_Type_create_darray.3	2007-06-23 12:48:18.000000000 -0500
@@ -98,7 +98,7 @@
 - MPI_DISTRIBUTE_NONE - Dimension not distributed. 
 .fi
 .sp
-The constant MPI_DISTRIBUTE_DFLT_DARG specifies a default distribution argument. The distribution argument for a dimension that is not distributed is ignored. For any dimension \fIi\fP in which the distribution is MPI_DISTRIBUTE_BLOCK, it erroneous to specify \fIarray_of_dargs[i]\fP \* \fIarray_of_psizes[i]\fP < \fIarray_of_gsizes[i]\fP.
+The constant MPI_DISTRIBUTE_DFLT_DARG specifies a default distribution argument. The distribution argument for a dimension that is not distributed is ignored. For any dimension \fIi\fP in which the distribution is MPI_DISTRIBUTE_BLOCK, it erroneous to specify \fIarray_of_dargs[i]\fP \fI\*\fP \fIarray_of_psizes[i]\fP < \fIarray_of_gsizes[i]\fP.
 .sp
 For example, the HPF layout ARRAY(CYCLIC(15)) corresponds to MPI_DISTRIBUTE_CYCLIC with a distribution argument of 15, and the HPF layout ARRAY(BLOCK) corresponds to MPI_DISTRIBUTE_BLOCK with a distribution argument of MPI_DISTRIBUTE_DFLT_DARG. 
 .sp
diff -urNad openmpi-1.2.3.orig/orte/tools/orterun/orterun.1 openmpi-1.2.3/orte/tools/orterun/orterun.1
--- openmpi-1.2.3.orig/orte/tools/orterun/orterun.1	2007-06-14 23:42:17.000000000 -0500
+++ openmpi-1.2.3/orte/tools/orterun/orterun.1	2007-06-23 15:38:22.000000000 -0500
@@ -22,26 +22,20 @@
 .PP
 Single Process Multiple Data (SPMD) Model:
 
-.B mpirun 
-.R [ options ] 
-.B <program>
-.R [ <args> ]
-.
+.B mpirun [ options ] 
+.B <program> [ <args> ]
+.P
 
 Multiple Instruction Multiple Data (MIMD) Model:
 
-.B mpirun
-.R [ global_options ]
+.B mpirun .R [ global_options ]
        [ local_options1 ]
-.B <program1>
-.R [ <args1> ] :
+.B <program1> .R [ <args1> ] :
        [ local_options2 ]
-.B <program2>
-.R [ <args2> ] : 
+.B <program2> .R [ <args2> ] : 
        ... : 
        [ local_optionsN ]
-.B <programN>
-.R [ <argsN> ]
+.B <programN> .R [ <argsN> ]
 .P
 
 Note that in both models, invoking \fImpirun\fR via an absolute path
